test/func/config/ttk-ttksim1/spec_files/system_config.json

```diff
@@ -77,6 +77,25 @@
       "hostnames": [],
       "prefix": "/sdk-out",
       "asynchronous": true
+    },
```

Comment:
Can we remove these apis mojaloop_simulator_inbound and scheme_adapter_outbound as you added another version of them?